user-guide/Basic_Functionality/Protocols_and_templates/Alarm_templates/Configuring_alarm_templates/Configuring_anomaly_detection_alarms.md

```diff
@@ -70,19 +70,27 @@ To configure anomaly detection alarms for specific parameters<!--RN 37171 + 3714
         > [!NOTE]
         > Any customized behavioral anomaly monitoring setup containing relative or absolute thresholds will be lost if you downgrade to DataMiner version 10.3.11 or older. These versions do not support this extended anomaly configuration, and the thresholds will be automatically determined by DataMiner Analytics, similar to the default *Smart* option<!--RN 37434-->.
 
-1. Click *Close* in the lower right corner to exit the *Anomaly Alarm Settings* pop-up window.
+1. From DataMiner 10.4.12/10.5.0 onwards<!-- RN 40837+41017 -->, in the *Proactive alarm setting* section, optionally configure the alarms that should be triggered based on [proactive cap detection](xref:Proactive_cap_detection):
```

Comment:
Add 'you can' optionally configure?

Reply:
I'd prefer not to, as instructions in a procedure are usually phrased as imperative sentences, so this would cause a bit of an inconsistency.